### PR TITLE
Update zoomus

### PR DESCRIPTION
**Pull request by lulor**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

As noted in #82016, the `uninstall` and `zap` stanzas still leave some files behind

### CODE REVIEW

File: Casks/zoomus.rb [hunk 1]
```diff
@@ -24,6 +24,7 @@
             login_item: 'ZoomOpener',
```
Comment (vladimyr) [inline Casks/zoomus.rb:24]:
```suggestion
            login_item: 'ZoomOpener',
            script:     {
                          executable:   '/usr/bin/defaults',
                          args:         ['delete', 'us.zoom.xos'],
                          must_succeed: false,
                          sudo:         true,
                        },
```

**Comment by vladimyr** [comment]
@lulor Please clean signal part too:

```diff
diff --git a/Casks/zoomus.rb b/Casks/zoomus.rb
index 8ba8c1c88c..ab9b552108 100644
--- a/Casks/zoomus.rb
+++ b/Casks/zoomus.rb
@@ -17,9 +17,7 @@ cask 'zoomus' do
   end
 
   uninstall quit:       'us.zoom.ZoomOpener',
-            signal:     [
-                          ['KILL', 'us.zoom.xos'],
-                        ],
+            signal:     ['KILL', 'us.zoom.xos'],
             pkgutil:    'us.zoom.pkg.videmeeting',
             login_item: 'ZoomOpener',
             delete:     [

```

File: Casks/zoomus.rb (hunk 1 above)
Comment (lulor) [reply Casks/zoomus.rb:22]: Thanks, I erroneously thought this was not necessary!

File: Casks/zoomus.rb [hunk 2]
```diff
@@ -32,12 +37,17 @@
                '~/.zoomus',
                '~/Desktop/Zoom',
                '~/Documents/Zoom',
+               '~/Library/Application Support/CloudDocs/session/containers/iCloud.us.zoom.videomeetings',
+               '~/Library/Application Support/CloudDocs/session/containers/iCloud.us.zoom.videomeetings.plist',
                '~/Library/Application Support/zoom.us',
                '~/Library/Caches/us.zoom.xos',
                '~/Library/Cookies/us.zoom.xos.binarycookies',
                '~/Library/Logs/zoom.us',
                '~/Library/Logs/zoominstall.log',
```
Comment (miccal) [inline Casks/zoomus.rb:46]: Please add:
```
'~/Library/Logs/ZoomPhone',
```

File: Casks/zoomus.rb [hunk 3]
```diff
@@ -17,13 +17,18 @@
...
             login_item: 'ZoomOpener',
+            script:     {
+                          executable:   '/usr/bin/defaults',
+                          args:         ['delete', 'us.zoom.xos'],
+                          must_succeed: false,
+                          sudo:         true,
+                        },
             delete:     [
                           '/Applications/zoom.us.app',
+                          '/Library/Internet Plug-Ins/ZoomUsPlugIn.plugin',
```
Comment (miccal) [inline Casks/zoomus.rb:31]: Please add:
```
'/Library/Logs/DiagnosticReports/zoom.us*',
```

Comment (lulor) [reply Casks/zoomus.rb:31]: Thanks!

**Pull request merged**